src/util/simplify_expr_struct.cpp

```diff
@@ -15,15 +15,13 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "pointer_offset_size.h"
 #include "std_expr.h"
 
-simplify_exprt::resultt<> simplify_exprt::simplify_member(const exprt &expr)
+simplify_exprt::resultt<>
+simplify_exprt::simplify_member(const member_exprt &expr)
 {
-  if(expr.operands().size()!=1)
-    return unchanged(expr);
-
   const irep_idt &component_name=
     to_member_expr(expr).get_component_name();
 
-  const exprt &op = expr.op0();
+  const exprt &op = expr.struct_op();
```

Comment:
Use `.compound()`, `struct_op` carries a comment that "it will go away" (repeats below)